components/tikv_util/src/log.rs

```diff
@@ -0,0 +1,49 @@
+/// Logs a critical level message using the slog global logger.
+#[macro_export]
+macro_rules! crit( ($($args:tt)+) => {
+    ::slog_global::crit!($($args)+)
+};);
+/// Logs a error level message using the slog global logger.
```

Comment:
Better to have some doc for usage and requirements. (min_specialization feature and slog_global dependency)